test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -32,6 +32,158 @@
     # Internal, profiler-specific ops
     ("profiler::_call_end_callbacks_on_jit_fut*", datetime.date(9999, 1, 1)),
     ("profiler::_record_function_enter", datetime.date(9999, 1, 1)),
+    ("tensorexpr::Group", datetime.date(2020, 9, 9)),
```

Comment:
This seems like a lot of unrelated changes? Was there a merge issue?

Reply:
yes, I think I did a bad merge. They were killed by 49c9994fb7 and probably I reintroduced them by mistake.
Killed them again in commit 609b3fdb8d